packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -105,6 +117,23 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// {@macro flutter.widgets.RawAutocomplete.initialValue}
   final TextEditingValue? initialValue;
 
+  /// If the options view overlay should be shown when the options returned from
+  /// [optionsBuilder] is empty.
+  ///
+  /// Set to true to display a custom options view on
+  /// first focus or to display a message if no options are found.
```

Comment:
What if you only want to do one but not the other, is that possible?

Reply:
Yes. In the end, it is up to the user to declare what they want to return in `optionsViewBuilder`, and they can choose to do both, none, or one but not the other.